src/mockserver.js

```diff
@@ -1,9 +1,12 @@
 var config = window.__karma__.config,
-	ui5config = (config && config.openui5) || {},
+	ui5config = (config && config.openui5.config) || {},
 	mockserverConfig = ui5config.mockserver || {};
 
 //workaround for crossroads/mockserver issue
 (function() {
+	if (typeof module === undefined) {
+		var module = '';
```

Comment:
`var original = module;`is normally handled like `var original = undefined;` if module was not defined before. What kind of environment are you using to execute the tests?